ehitchhiking-ui/src/app/shared/components/active-trips-modal/active-trips-modal.component.sass

```diff
@@ -0,0 +1,65 @@
+@import '../../../../styles/fonts-constants'
```

Comment:
copy paste from the HistoryModalComponent

Reply:
fixed